_locales/en/messages.json

```diff
@@ -2645,20 +2661,19 @@
     "message": "Channels"
   },
   "addChannel": {
-    "message": "Add channel"
+    "message": "Join Open Group"
   },
   "enterChannelURL": {
-    "message": "Enter Channel URL"
+    "message": "Enter Open Group URL"
   },
   "channelUrlPlaceholder": {
-    "message": "https://yourchannel.lokinet.org"
+    "message": "https://myopengroup.lokinet.org"
```

Comment:
Om mobile this is the chat.lokinet.org url which is displayed. Mayne we should make this the default too.

Reply:
Done